pathfinder/api/rest.py

```diff
@@ -107,6 +108,14 @@ def get(self, token_network_address: str):
 
         return {'result': paths}, 200
 
+class InfoResource(PathfinderResource):
+    def get(self):
+        ip = socket.gethostbyname(socket.gethostname()) ## this is the internal IP address, which is a feature that will be scoped out soon anyway
```

Comment:
also please run flake8 on PFS and MS. This comment line will hit on flake8. Needs an additional space before #

Reply:
Yes, there is https://github.com/raiden-network/raiden-pathfinding-service/issues/88

No need to do this in this PR